src/onnx/parse_dynamicquantizelinear.cpp

```diff
@@ -0,0 +1,151 @@
+uint8 is supported.
+* rounding to nearest ties to even.
+
+Version
+This version of the operator has been available since version 11 of the default ONNX operator set.
+
+Inputs
+x : T1
+Input tensor
+
+Outputs
+y : T2
+Quantized output tensor
+
+y_scale : tensor(float)
+Output scale. It's a scalar, which means a per-tensor/layer quantization.
+
+y_zero_point : T2
+Output zero point. It's a scalar, which means a per-tensor/layer quantization.
+
+Type Constraints
+T1 : tensor(float)
+Constrain 'x' to float tensor.
+
+T2 : tensor(uint8)
+Constrain 'y_zero_point' and 'y' to 8-bit unsigned integer tensor.
+*/
+
+struct parse_dynamicquantizelinear : op_parser<parse_dynamicquantizelinear>
+{
+    std::vector<op_desc> operators() const { return {{"DynamicQuantizeLinear"}}; }
+
+    std::vector<instruction_ref> parse(const op_desc& /*opd*/,
+                                       const onnx_parser& /*parser*/,
+                                       const onnx_parser::node_info& info,
+                                       const std::vector<instruction_ref>& args) const
+    {
+        auto x       = args[0];
+        auto x_shape = x->get_shape();
+        if(x_shape.dynamic())
+            MIGRAPHX_THROW("DYNAMICQUANTIZELINEAR: dynamic shapes are not supported");
+
+        auto x_type = x_shape.type();
+        // 1. Computing y_scale
+        auto l0 = info.add_literal(migraphx::literal{migraphx::shape{x_type}, {0}});
+        // DynamicQuantizeLinear only has uint8 quantization
+        auto q_max = info.add_literal(
+            migraphx::literal{migraphx::shape{x_type}, {std::numeric_limits<uint8_t>::max()}});
+        auto q_min = info.add_literal(
+            migraphx::literal{migraphx::shape{x_type}, {std::numeric_limits<uint8_t>::min()}});
+        auto x_reshape = x;
+        if(not(x_shape.lens().size() == 1))
+        {
+            x_reshape = info.add_instruction(
+                migraphx::make_op("reshape", {{"dims", {x_shape.elements()}}}), x);
+        }
+        // maximum(0, max(x))
+        auto topk_max = info.add_instruction(
+            migraphx::make_op("topk", {{"axis", 0}, {"k", 1}, {"largest", true}}), x_reshape);
+        auto max_x =
+            info.add_instruction(migraphx::make_op("get_tuple_elem", {{"index", 0}}), topk_max);
+        max_x = info.add_instruction(migraphx::make_op("max"), l0, max_x);
+
+        // minimum(0, min(x))
+        auto topk_min = info.add_instruction(
+            migraphx::make_op("topk", {{"axis", 0}, {"k", 1}, {"largest", false}}), x_reshape);
+        auto min_x =
+            info.add_instruction(migraphx::make_op("get_tuple_elem", {{"index", 0}}), topk_min);
+        min_x = info.add_instruction(migraphx::make_op("min"), l0, min_x);
+
+        // y_scale = (maximum(0, max(x)) - minimum(0, min(x))) / (qmax - qmin)
+        auto sub0    = info.add_instruction(migraphx::make_op("sub"), max_x, min_x);
+        auto y_scale = info.add_instruction(migraphx::make_op("div"), sub0, q_max);
```

Comment:
Optional. `(q_max - q_min)` instead of just `q_max` on line 130.

Reply:
Excuse me, this is a compile time step, not a new (additional) compute instruction. Hence the above suggestion. Sorry about any confusion.